### PR TITLE
Translate to_sentence for connecting titles

### PR DESCRIPTION
**Pull request by 1pretz1**
Previously, to_sentence was not using translations and so injecting English connecting words like 'and' into foreign translations.

## Before

Screenshot 2022-10-04 at 17 05 36

## After

Screenshot 2022-10-04 at 17 05 27

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: spec/models/person_spec.rb [hunk 1]
```diff
@@ -54,6 +54,13 @@
     it "combines the titles into a sentence" do
       expect(person.current_roles_title).to eq("Prime Minister and First Lord of the Treasury")
     end
+
+    it "combines the titles into a sentence for translated content" do
```
Comment (JonathanHallam) [inline spec/models/person_spec.rb:58]: odd question, would it be worth putting the text in english to make it clearer what's actually being tested something like `one و two`

Comment (1pretz1) [reply spec/models/person_spec.rb:58]: I see your point for readability, however the current setup is more realistic, as we shouldn't have a translated page with English words. I have added a comment above the test which should make the test more obvious.

Comment (JonathanHallam) [reply spec/models/person_spec.rb:58]: thanks :)

**Review by JonathanHallam — APPROVED** [review]
👍

**Review by gclssvglx — APPROVED** [review]
Nice work @1pretz1 👍

**Pull request merged**